src/test/ui/moves/use_of_moved_value_copy_suggestions.rs

```diff
@@ -1,29 +1,23 @@
-// run-rustfix
```

Comment:
Can you revert this?

Reply:
I can't cuz else the tests fail because there was a diagnostic item even after the fix was applied (or is there a way to tell the test suit that it's okay if there are warnings after the fix?)

Reply:
Oh, okay I just thought it would be weird to have a meaningless and effectless `where` after a function definition or smth like this

Reply:
Sure, but it will introduce new warnings in previously working code regardless. Pair this together with `deny(warnings)`, and you might break code in the wild.

Reply:
Okay, that's a fair point, i'll remove change